www/vonm10/cv09/admin/users.php

```diff
@@ -0,0 +1,53 @@
+<?php require_once __DIR__ . '/../db/UsersDB.php'; ?>
+
+<?php
+session_start();
+if ($_SESSION['admin'] != 3) {
+    header('Location: /./~vonm10/cv06/index.php');
+}
+
+$usersDB = new UsersDB();
+$userRecords = $usersDB->fetchAll();
+
+if (!empty($_POST)) {
+    $usersDB->updateRole($_POST['button'], $_POST['newRole']);
+}
+
+?>
+
+
+
+
+<?php require __DIR__ . '/../incl/header.php'; ?>
+<h1>Users</h1>
+<form method="POST">
+    <table border="1">
+        <tr>
+            <th>id</th>
+            <th>email</th>
+            <th>password</th>
+            <th>role</th>
+            <th>new role</th>
+            <th>confirm</th>
+        </tr>
+        <?php foreach ($userRecords as $userRecord) : ?>

```

Comment:
pro kazdy radek uzivatele ma byt jeden formular

Reply:
To je zvlastni, pri testovani mi to fungovalo, ale ted se opravneni pri odeslani nemeni

Reply:
Melo by byt opraveno